eos/parameters/hadronic-matrix-elements.yaml

```diff
@@ -450,6 +494,7 @@ groups:
           unit:     'GeV'
       # form factor parameters for B->K for [BCL:2008A] parametrization
       'B->K::f_+(0)@BCL2008' :
+          alias_of: [ 'B_d->K_d::f_+(0)', 'B_d->Kbar_d::f_+(0)', 'B_d->K_u::f_+(0)','B_d->Kbar_u::f_+(0)', 'B_u->K_d::f_+(0)', 'B_u->Kbar_d::f_+(0)', 'B_u->K_u::f_+(0)','B_u->Kbar_u::f_+(0)' ]
```

Comment:
Hmm. I'm not sure that I am happy to mix the BCL2008 parameters with the QCDF parameters.

Reply:
We can duplicate the parameter of course, but these aliases make sense in any case, no?

Reply:
Ah, now that I see ``[parameters] Add q2 = 0 value for the B->K form factor``, I notice that you have already duplicated it!

Reply:
I will mark this comment as resolved.